src/main/resources/userEmailTemplate-courseJoin.html

```diff
@@ -3,9 +3,9 @@
 ${joinFragment}
 
 <p>
-    If you encounter any problems when using the system, you can email TEAMMATES support team at ${supportEmail}.
+  If you encounter any problems when using the system, you can email TEAMMATES support team at ${supportEmail}.
 </p>
 
 <p>Regards,
```

Comment:
Let's put a line break after `<p>`. Applies to all occurrences of `<p>Regards,`.